desktop/cmp/input/ButtonGroupInput.js

```diff
@@ -49,6 +50,33 @@ ButtonGroupInput.hasLayoutSupport = true;
 //----------------------------------
 // Implementation
 //----------------------------------
+class Model extends HoistInputModel {
+
+    blur() {
+        this.enabledButtons[0]?.blur();
+    }
+
+    onBlur = (e) => {
+        const noBtnsFocused = this.enabledButtons.every(it => !it.focused);
+
+        if (noBtnsFocused) {
+            // noting blur for entire button group, not a single button within
+            this.noteBlurred();
+        }
+    };
+
+    focus() {
+        this.enabledButtons[0]?.focus();
+    }
+
+    get enabledButtons() {
+        const btns = ReactDOM.findDOMNode(this.domRef.current)
```

Comment:
Yikes!  This seems like a mistake.  This should be built-in.  Thought domRef was already returning a ref to the dom node1

Reply:
Let's go over these implementations.  Not sure I fully understand this, but I see that it is complex.